arches_lingo/src/arches_lingo/components/widgets/ConceptResourceSelectWidget/components/ConceptResourceSelectWidgetEditor.vue

```diff
@@ -0,0 +1,235 @@
+                .filter((option) => {
+                    return newVal?.includes(option.id);
+                })
+                .map((option) => {
+                    return {
+                        resourceId: option.id,
+                        ontologyProperty: "",
+                        inverseOntologyProperty: "",
+                    };
+                });
+        }
+    },
+);
+
+const searchResultsCurrentCount = computed(() => options.value.length);
+
+function clearOptions() {
+    options.value = props.initialValue || [];
+}
+
+function onFilter(event: MultiSelectFilterEvent) {
+    clearOptions();
+    getOptions(1, event.value);
+}
+
+async function getOptions(page: number, filterTerm?: string) {
+    try {
+        isLoading.value = true;
+        const parsedResponse = await fetchConceptResources(
+            filterTerm || "",
+            itemSize,
+            page,
+            props.scheme,
+            props.exclude,
+        );
+
+        parsedResponse.data.forEach((option: SearchResultItem) => {
+            option.label = getItemLabel(
+                option,
+                ENGLISH.code,
+                ENGLISH.code,
+            ).value;
+        });
+        if (page === 1) {
+            options.value = parsedResponse.data;
+        } else {
+            options.value = [...options.value, ...parsedResponse.data];
+        }
+
+        searchResultsPage.value = parsedResponse.current_page;
+        searchResultsTotalCount.value = parsedResponse.total_results;
+    } catch (error) {
+        fetchError.value = (error as Error).message;
+    } finally {
+        isLoading.value = false;
+    }
+}
+
+async function onLazyLoadResources(event?: VirtualScrollerLazyEvent) {
+    if (isLoading.value) {
+        return;
+    }
+
+    if (
+        // if we have already fetched all the resources
+        searchResultsTotalCount.value > 0 &&
+        searchResultsCurrentCount.value >= searchResultsTotalCount.value
+    ) {
+        return;
+    }
+
+    if (
+        // if the user has NOT scrolled to the end of the list
+        event &&
+        event.last < searchResultsCurrentCount.value - 1
+    ) {
+        return;
+    }
+
+    if (
+        // if the dropdown is opened and we already have data
+        !event &&
+        searchResultsCurrentCount.value > 0
+    ) {
+        return;
+    }
+
+    await getOptions((searchResultsPage.value || 0) + 1);
+}
+
+function resolver(e: FormFieldResolverOptions) {
+    validate(e);
+}
+
+function validate(e: FormFieldResolverOptions) {
+    console.log("validate", e);
+}
+</script>
+
+<template>
+    <Message
+        v-if="fetchError"
+        severity="error"
+    >
+        {{ fetchError }}
+    </Message>
+    <FormField
+        v-else
+        ref="formFieldRef"
+        v-slot="$field"
+        :name="props.nodeAlias"
+        :initial-value="props.initialValue?.map((concept) => concept.id)"
+        :resolver="resolver"
+    >
+        <MultiSelect
```

Comment:
this still need a fair amount of work. I'd recommend chatting with @aarongundel about the intricacies involving multiselect.

Essentially, if a user selects several values, the text runs off of the page.
Additionally, selecting values and then filtering the list/reloading the component results in those selected values being `null`.